index.html

```diff
@@ -161,3 +187,10 @@ <h2 class="text-center">About Us</h2>
     </div>
   </div>
 </section>
+
+<script>
+  var options = {
+    valueNames: ['name', 'description']
```

Comment:
This searches names and descriptions currently, but for providing meta search, it is possible to add another html element for each resource with a hidden display style. This makes it possible to "tag" resources according to categories like "green-card holder" or "permanent-resident", etc, to provide a more flexible search experience.